src/webgpu/api/operation/texture_view/texture_component_swizzle.spec.ts

```diff
@@ -0,0 +1,681 @@
+export const description = `
+Operational tests for the 'texture-component-swizzle' feature.
+
+Test that:
+* when the feature is on, swizzling is applied correctly.
+
+TODO:
+* test stencil aspect of depth-stencil formats
+* test texture_depth_xxx with textureSample
```

Comment:
I believe texture_depth_xx are actually tested with textureSample in this PR. Am I missing something?

Reply:
@greggman gentle ping in case it was missed.